pinot-core/src/test/java/org/apache/pinot/core/startree/v2/BaseStarTreeV2Test.java

```diff
@@ -141,7 +141,9 @@ public void setUp()
 
     // Randomly build star-tree using on-heap or off-heap mode
     BuildMode buildMode = RANDOM.nextBoolean() ? BuildMode.ON_HEAP : BuildMode.OFF_HEAP;
-    new MultipleTreesBuilder(Collections.singletonList(starTreeV2BuilderConfig), indexDir, buildMode).build();
+    MultipleTreesBuilder builder = new MultipleTreesBuilder(Collections.singletonList(starTreeV2BuilderConfig), indexDir, buildMode);
+    builder.build();
+    builder.close();
```

Comment:
MultipleTreesBuilder is not a closeable, so I am not sure why we need to call close here.

Reply:
ditto.

Reply:
Can you post the failure message? What does it complain about?

Reply:
ditto, it reports:
`java.io.IOException: Unable to delete file: C:\Users\yu.xue\AppData\Local\Temp\BaseStarTreeV2Test\testSegment\v3\columns.psf`